reactioncommerce/reaction
# [Marketplace] Order Filtering By Status

Resolves #2357 

*Description*
Users with permissions to view orders dashboard should be able to quickly filter the list of orders by the order status.

*How to test*
- Clone branch
- Create multiple orders
- Have some orders in different stages of the order workflow ["New", "Approved", "Captured", "Completed", "Cancelled"]
- Use filters observe orders display according to filter
- Click on left-most icon to clear filter [Observe all your orders displayed]
